pkg/pair/pair.go

```diff
@@ -233,7 +273,7 @@ func weighSwiss(members *entity.UnpairedPoolMembers, i int, j int) int64 {
 
 	// Subtract the spread difference for swiss, since we would like to pair players
 	// that have similar records but large differences in spread.
-	spreadDiffWeight := -int64(utilities.Abs(p1.Spread-p2.Spread))
+	spreadDiffWeight := -int64(utilities.Abs(p1.Spread - p2.Spread))
```

Comment:
`gofmt` ?

Reply:
Must be, I don't remember changing that.